x-pack/plugins/observability_solution/apm/server/lib/helpers/get_document_sources.ts

```diff
@@ -240,7 +247,7 @@ const mapToSources = (
       documentType,
       rollupInterval,
       hasDocs,
-      hasDurationSummaryField: allHaveDurationSummary,
+      hasDurationSummaryField: hasDocs && allHaveDurationSummary,
```

Comment:
For consistency, it doesn't make sense to return `true` in `hasDurationSummaryField` when `hasDocs` is `false`